layers/meta-resin-edison/recipes-core/images/resin-image.bbappend

```diff
@@ -55,7 +55,7 @@ deploy_bundle() {
     cp -rL ${DEPLOY_DIR_IMAGE}/u-boot-edison.bin ${DEPLOY_DIR_IMAGE}/resin-edison/
     cp -rL ${DEPLOY_DIR_IMAGE}/u-boot-edison.img ${DEPLOY_DIR_IMAGE}/resin-edison/
     cp -rL ${DEPLOY_DIR_IMAGE}/u-boot-envs ${DEPLOY_DIR_IMAGE}/resin-edison/
-    cp -rL ${DEPLOY_DIR_IMAGE}/resin-image-edison.ext4 ${DEPLOY_DIR_IMAGE}/resin-edison/
+    cp -rL ${DEPLOY_DIR_IMAGE}/resin-image-edison.hostapp-ext4 ${DEPLOY_DIR_IMAGE}/resin-edison/
```

Comment:
I wonder if we could use e.g. RESIN_ROOT_FSTYPE here?

Reply:
Done @willnewton 
Please re-review